scripts/scil_freewater_priors.py

```diff
@@ -2,20 +2,11 @@
 # -*- coding: utf-8 -*-
 
 """
-Compute the axial (para_diff) and mean (iso_diff) diffusivity priors for
-freewater.
+Synonym for scil_NODDI_priors.py
```

Comment:
I don't know why we would reduce the explanation here. I would add your sentence after what was previously here.

Reply:
Because it does not show anywhere. If you do `scil_freewater_priors -h`, it prints the expanation of scil_NODDI_priors.

Reply:
Ah I see!